Tools/MobileMiniBrowser/MobileMiniBrowserFramework/AppDelegate.m

```diff
@@ -61,7 +61,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
         viewController.initialURL = url;
 
     if (!self.window)
+// FIXME: rdar://151039019 (Replace using 'init' in AppDelegate.m with non-deprecated method)
+ALLOW_DEPRECATED_DECLARATIONS_BEGIN
         self.window = [[UIWindow alloc] init];
+ALLOW_DEPRECATED_DECLARATIONS_END
```

Comment:
Looks like this will need to use `#pragma` directly:

```suggestion
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
        self.window = [[UIWindow alloc] init];
#pragma clang diagnostic pop
```

Reply:
done